src/__experimental__/components/help/help.component.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon from '../../../components/icon';
+import tagComponent from '../../../utils/helpers/tags';
+import StyledHelp from './help.style';
+
+class Help extends React.Component {
+  static propTypes = {
+    className: PropTypes.string,
+    children: PropTypes.string,
+    type: PropTypes.string,
+    tooltipPosition: PropTypes.string,
+    tooltipAlign: PropTypes.string,
+    href: PropTypes.string
+  };
+
+  static defaultProps = {
+    tooltipPosition: 'top',
+    tooltipAlign: 'center'
+  }
+
+  get mainClasses() {
+    return classNames(
+      'carbon-help',
```

Comment:
Do we need class names?

Reply:
I don't think it hurts to keep it, making wider changes to Help feels out of scope for the Checkbox piece, it's pretty much a straight copy of the existing component, with the small tweak I needed to make the Checkbox work

Reply:
If the class names are unused then they probably shouldn't be in there.

Reply:
You have access to the `href` prop in the styled component. I think you can delete all classes.

Reply:
Removed